docs/team/samuelfoo.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Samuel Foo's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```

Comment:
This seems to be the template portfolio, instead of your own. Are we sure this is relevant? Perhaps we don't need to finish writing the whole portfolio just yet.

Reply:
Agreed. Removed details about AddressBook and kept only the headers to fill in as a template.